### PR TITLE
Feat: add ZCS hybrid triphase inverter

### PR DESCRIPTION
**Pull request by dadna**

### CODE REVIEW

File: devices/test.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
+const a = [
```
Comment (claudio-petrini) [inline devices/test.js]: We can't afford to call variables like this

File: devices/test.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+}
+
+const signedUnitsMap = {
+    1: "ca2Convertion",
+    0.1: "decimalSignDivider",
+    0.01: "centesimalSignDivider",
+    0.001: "millesimalSignDivider",
+}
+
+const o = {
```
Comment (claudio-petrini) [inline devices/test.js]: Same thing goes for this line

File: devices/test.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+        if (!prev.hasOwnProperty(register)) {
+            prev[register] = { address }
+            if (dataType === 'i16') prev[register].post = signedUnitsMap[unit];
+            else if (dataType === 'u16') prev[register].post = unsignedUnitsMap[unit];
+        }
+        return prev;
+    }, {})
+}
+
+console.log(JSON.stringify(o, null, 4))
```
Comment (claudio-petrini) [inline devices/test.js]: What kind test should be this 🤔

**Review by claudio-petrini — CHANGES_REQUESTED** [review]
@dadna : looks nice but I would suggest to use meaningful variable names in `test.js` (it would be nice to have an actual test as well) other than test the actual device.

**Comment by dadna** [comment]
sorry my bad, that was a local file for debugging purposes. i deleted it

**Comment by claudio-petrini** [comment]
@dadna : if you don't update the `devices.js` ([ref](https://github.com/Instathings/modbus-herdsman-converters/blob/triphase_inverter/devices.js)) file as well the new file won't be used.

**Comment by claudio-petrini** [comment]
Tests are not passing [ref](https://github.com/Instathings/modbus-herdsman-converters/actions/runs/4636620941/jobs/8204687832?pr=20#step:4:11)

**Pull request merged**